modules/p2/ding_serendipity_fallback/ding_serendipity_fallback.module

```diff
@@ -44,7 +44,12 @@ function ding_serendipity_fallback_ting_object_serendipity_add($context) {
 
   if (!empty($query)) {
     try {
-      $results = ding_serendipity_do_search($query, array('limit' => 8));
+      /** @var \Ting\Search\TingSearchRequest $sal_query */
+      $sal_query = ting_start_query();
```

Comment:
Is the DocBlock needed with proper type hinting for `ting_start_query()`?

Reply:
Nope, it was a leftover, removing it.